_test/test_multifit/test_multifit_horace_dnd_1.m

```diff
@@ -0,0 +1,182 @@
+            [wfit_single1,fitpar_single1] = mss.fit();
+            
+            mss = multifit_sqw(this.w2data);
+            mss = mss.set_fun(@sqw_bcc_hfm,  [5,5,1,10,0]);  % set foreground function(s)
+            mss = mss.set_free([0,1,1,1,1]); % set which parameters are floating
+            mss = mss.set_bfun(@linear_bkgd, [0,0]); % set background function(s)
+            [wfit_single2,fitpar_single2] = mss.fit();
+            
+            mss = multifit_sqw(this.win);
+            mss = mss.set_fun(@sqw_bcc_hfm,  [5,5,1,10,0]);  % set foreground function(s)
+            mss = mss.set_free([0,1,1,1,1]); % set which parameters are floating
+            mss = mss.set_bfun(@linear_bkgd, [0,0]); % set background function(s)
+            [wfit_single12,fitpar_single12] = mss.fit();
+            
+            assertTrue(equal_to_tol([wfit_single1,wfit_single2],wfit_single12),'fit_sqw workspaces not working');
+            assertTrue(equal_to_tol([fitpar_single1,fitpar_single2],fitpar_single12),'fit_sqw fitting not working')
+            
+%             % Test against saved or store to save later
+%             this=save_or_test_variables(this,wfit_single1,wfit_single2,wfit_single12);
+        end
+        
+        % ------------------------------------------------------------------------------------------------
+        function this=DISABLED_test_fit_test_fit_array_of_datasets_2(this)
+            % Example of fitting single datasets, and independently fitting an array of datasets
+            % This exercises multifit_sqw for single datasets and compares with fit_sqw for an array of datasets
+            
+            % *** CODE NEEDS TO BE EDITED ONCE FIT_SQW CREATED
+            
+            [wfit_single12,fitpar_single12]=fit_sqw(this.win, @sqw_bcc_hfm, [5,5,1,10,0], [0,1,1,1,1], @linear_bkgd, [0,0]);
+            
+            [wfit_single1,fitpar_single1]=multifit_sqw(this.w1data, @sqw_bcc_hfm, [5,5,1,10,0], [0,1,1,1,1], @linear_bkgd, [0,0]);
+            [wfit_single2,fitpar_single2]=multifit_sqw(this.w2data, @sqw_bcc_hfm, [5,5,1,10,0], [0,1,1,1,1], @linear_bkgd, [0,0]);
+            
+            assertTrue(equal_to_tol([wfit_single1,wfit_single2],wfit_single12),'fit_sqw not working for dataset');
+            assertTrue(equal_to_tol([fitpar_single1,fitpar_single2],fitpar_single12),'fit_sqw not working for fit parameters');
+            
+            fitpar_single1.corr=[];
+            fitpar_single2.corr=[];
+            fitpar_single12(1).corr=[];
+            fitpar_single12(2).corr=[];
+            
+%             tol = this.tol;
+%             this.tol = -1;           
+%             this=save_or_test_variables(this,fitpar_single1,fitpar_single2,fitpar_single12);
+%             this.tol=tol;
+        end
+        
+        % ------------------------------------------------------------------------------------------------
+        function this=DISABLED_test_fit_array_of_datasets_3(this)
```

Comment:
why is this disabled

Reply:
the name doesn't tell me what it's testing and how it's different to `..._array_of_datasets` or `..._array_of_datasets_2`